.github/workflows/receive-pr.yml

```diff
@@ -0,0 +1,43 @@
+name: Receive Pull Request
+
+on: [pull_request]
+
+jobs:
+  build-and-test:
+    name: Build And Test
+    runs-on: windows-2019
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 3.1.301
+    - name: Restore nHapi
+      run: |
+        dotnet restore nHapi.sln --configfile build\.nuget\NuGet.config
+        dotnet restore Hl7Models.sln --configfile build\.nuget\NuGet.config
+    - name: Build nHapi
```

Comment:
as a dev, is the preferred way to assure build & test to use the psake scripts in the repo? If so, we could call into those from these steps? That way, the pipeline and local dev processes are the same. What do you think?

Reply:
I agree, I would like to harmonise the build scripts and GitHub actions so that GitHub actions run the same commands we would run locally.
However, I'm not certain the psake should be the tool for the job - so until that has been considered I chose to leave the existing psake build scripts.

Reply:
sweet, sounds good